JSDemos/Demos/CheckBox/Overview/Angular/description.md

```diff
@@ -1,9 +1,13 @@
-When you add a CheckBox to an application, set its [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) property. The CheckBox can have the following states: 
+When you add the CheckBox component to an application, specify its [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) property. The CheckBox can have the following states: 
 
-- Checked (**value** is **true**).
-- Unchecked (**value** is **false**).
-- Undetermined (**value** is **undefined**).
+- Checked ([value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) is `true`).
+- Unchecked ([value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) is `false`).
+- Indeterminate ([value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) is `null` or `undefined`).
 
-To handle value change, use two-way binding to bind the **value** property to a component property. In this demo, the **value** properties of the "Handle value change" and "Disabled" CheckBox components are bound to the same component property. Click the first CheckBox to see how it affects the second one. You can also use the [onValueChanged](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#onValueChanged) property to handle value change, for example, if you need to use the previous value.
+The component supports three state mode. In this demo, the [enableThreeStateBehavior](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#enableThreeStateBehavior) property of the "Three state mode" CheckBox is enabled. You can cycle through this CheckBox states in the following order:
+
+*Indeterminate → Checked → Unchecked → Indeterminate →  ...*
+
+To handle value change, use two-way binding to bind the [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) property to a component property. In this demo, the [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) properties of the "Handle value change" and "Disabled" CheckBox components are bound to the same component property. Click the first CheckBox to see how it affects the second one. You can also use the [onValueChanged](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#onValueChanged) property to handle value change, for example, if you need to use the previous value.
```

Comment:
> To which component property should I bind the Value property?
It depends on a property you want to change while checking the checkbox. In the demo, the checkbox **value** is bound to another checkbox **value**, so when you check the first, the second is checked too simultaneously. But the Checkbox **value** may be bound to any other component property, it can enable/disable Chart export, show/hide Popup, etc. That's why I write "**a** component property".
Maybe this version is better:
> To handle value changes, use two-way binding of the **value** property to a component property. The value of the component property depends on the **value** of the CheckBox.
Or even this:
> To handle value changes, use two-way binding. Bind the **value** property to a component property that you want to change when users click the CheckBox.

Reply:
Yeah, now it has more sense. Thank you.

> To handle value changes, use two-way binding to bind the [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) property to a component property. The value of the component property depends on the value of the CheckBox. In this demo, the [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) properties of the "Handle value change" and "Disabled" CheckBox components are bound to the same component property. Click the first CheckBox to see how it affects the second one. You can also use the [onValueChanged](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#onValueChanged) property to handle value changes. For example, this technique can be useful if you need to use the previous value.